identity/src/main/java/com/stripe/android/identity/ui/IdentityTheme.kt

```diff
@@ -0,0 +1,102 @@
+package com.stripe.android.identity.ui
+
+import android.content.res.Resources
+import android.os.Build
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.remember
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalInspectionMode
+import androidx.compose.ui.platform.LocalLayoutDirection
+import com.google.accompanist.themeadapter.material.createMdcTheme
+import com.stripe.android.uicore.LocalColors
+import com.stripe.android.uicore.LocalShapes
+import com.stripe.android.uicore.LocalTypography
+import com.stripe.android.uicore.StripeThemeDefaults
+import java.lang.reflect.Method
+
+/**
+ * IdentityTheme tries to read theme values from hosting app's context. Then adapt the values with
+ * CompositionLocalProviders required by StripeTheme.
+ */
+@Composable
+internal fun IdentityTheme(content: @Composable () -> Unit) {
```

Comment:
Here's how connections is doing it in case you find it useful - https://github.com/stripe/stripe-android/blob/21b4ecc699c1591aa56dbe48ff2d3de0596e4240/financial-connections/src/main/java/com/stripe/android/financialconnections/ui/theme/StripeThemeForConnections.kt#L9